src/rgw/rgw_tag.h

```diff
@@ -15,9 +15,16 @@ class RGWObjTags
  protected:
   using tag_map_t = boost::container::flat_map <std::string, std::string>;
   tag_map_t tag_map;
+
+  uint32_t max_obj_tags{10};
+  uint32_t max_tag_key_size{128};
+  uint32_t max_tag_val_size{256};
```

Comment:
can these last two stay `static constexpr` if there's no way to customize them? that way don't add to sizeof(RGWObjTags)